File: zantt-api-backend/src/main/java/com/zantt/entities/TaskEntity.java
package com.zantt.entities;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "tasks")
public class TaskEntity {
    @Id
    @Column(name = "task_id")
    private String taskId;
    @Column(name = "project_id")
    private String projectId;
    @Column(name = "title")
    private String title;

    public String getTaskId() {
        return taskId;
    }

    public void setTaskId(String taskId) {
        this.taskId = taskId;
    }

    public String getProjectId() {
        return projectId;
    }

    public void setProjectId(String projectId) {
        this.projectId = projectId;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }
}

File: zantt-api-backend-net/Zantt/Entities/TaskEntity.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Zantt.Entities;

[Table("tasks")]
public class TaskEntity
{
    [Required]
    [Column("project_id")]
    [StringLength(255)]
    public string ProjectId { get; set; } = Guid.NewGuid().ToString();

    [Key]
    [Column("task_id")]
    [StringLength(255)]
    public string TaskId { get; set; } = Guid.NewGuid().ToString();

    [Required]
    [Column("title")]
    [StringLength(255)]
    public string Title { get; set; } = "";

    [Required]
    [Column("created_time")]
    public DateTime CreatedTime { get; set; } = DateTime.Now;
}
